introducing newsletter (#5468)
unlock-protocol/unlock: unlock-protocol.com/blog/introducing-newsletter.md

---
title: Newsletters
subTitle: Newsletters are memberships! Easily start collecting members with their email addresses using Unlock
authorName: Julien Genestoux
publishDate: December 12, 2019
description: Unlock lets you easily collect email addresses from members to create your own paid-for mailing list, using Ethereum and Non Fungible Tokens
---

There are many kinds of memberships. Going back to the origins of the internet, newsletters have always been a popular style of membership. Today, we're excited to show you how you can easily use a lock to create your own newsletter membership with Unlock!

# The Defiant

> A couple of weeks ago, while at DevCon in Osaka, I was introduced to [Camila](https://twitter.com/camirusso) who publishes one of the most insightful newsletters about the DeFi movement. I told her about Unlock and the fact that we [tokenize memberships](/tokenizing-memberships/), we quickly agreed that she needed to provide the ability to join her mailing list using crypto.

Reviewer: ```suggestion
A couple of weeks ago, while at DevCon in Osaka, I was introduced to [Camila](https://twitter.com/camirusso) who publishes one of the most insightful newsletters about the DeFi movement. When I told her about Unlock and the fact that we [tokenize memberships](/tokenizing-memberships/), we quickly agreed that she needed to provide the ability to join her mailing list using crypto.
```

> Today, that's possible! you can [become a member to The Defiant](https://bit.ly/the-defiant). If you go fast, you can join the [OG membership](https://etherscan.io/address/0xFA7001A0310B5E69B7b95B72aeBaA66C72E084bf) (limited to 100 members) for 70 DAI, or join her [regular membership](https://etherscan.io/address/0x43154efc9cb33c80833c0dec1e15bb9cfc1275e5) for the next year, at 100 DAI!

Reviewer: ```suggestion
Today, that's possible! You can [become a member of The Defiant](https://bit.ly/the-defiant) using DAI. If you join quickly, you can become an [OG member](https://etherscan.io/address/0xFA7001A0310B5E69B7b95B72aeBaA66C72E084bf) (limited to 100 members) for 70 DAI. If you miss the OG perk, you can still join her [regular membership](https://etherscan.io/address/0x43154efc9cb33c80833c0dec1e15bb9cfc1275e5) for the next year, at 100 DAI!
```

> Of course, like any other key, The Defiant membership keys are Non Fungible Tokens and your favorite crypto wallet should show them. Here's for example, my [OG membership on OpenSea](https://opensea.io/assets/0xFA7001A0310B5E69B7b95B72aeBaA66C72E084bf/1).

Reviewer: ```suggestion
Of course, like any other key, The Defiant membership keys are Non Fungible Tokens and your favorite crypto wallet should show them. For example, here's my [OG membership on OpenSea](https://opensea.io/assets/0xFA7001A0310B5E69B7b95B72aeBaA66C72E084bf/1).
```

> # How does it work

Reviewer: ```suggestion
# How It Works
```

> First, the creator has deployed their own lock on the Ethereum chain. Once you are on the newsletter landing page, you can enter your email address. As you submit, the address is saved inside the metadata of your token.

Reviewer: ```suggestion
First, the creator deploys their own lock on the Ethereum chain. Once you are on the newsletter landing page, you can enter your email address. As you submit, the address is saved inside the metadata of your token.
```
